openedx/core/djangoapps/cors_csrf/helpers.py

```diff
@@ -3,7 +3,7 @@
 
 import contextlib
 import logging
-import six.moves.urllib.parse  # pylint: disable=import-error
+import urllib.parse  # pylint: disable=import-error
```

Comment:
```suggestion
import urllib.parse
```
This shouldn't be a problem anymore.